pkg/controller/cyclenoderequest/transitioner/transitions.go

```diff
@@ -439,14 +437,14 @@ func (t *CycleNodeRequestTransitioner) transitionCordoning() (reconcile.Result,
 			// Try to send the trigger, if is has already been sent then this will
 			// be skipped in the function. The trigger must only be sent once
 			if err := t.sendPreTerminationTrigger(node); err != nil {
-				return t.transitionToHealing(errors.Wrapf(err, "failed to send pre-termination trigger, %s is still cordononed", node.Name))
+				t.rm.LogEvent(t.cycleNodeRequest, "PreTerminationTriggerFailed", "failed to send pre-termination trigger to %v, err: %v", node.Name, err)
 			}
 
 			// After the trigger has been sent, perform health checks to monitor if the node
 			// can be terminated. If all checks pass then it can be terminated.
 			allHealthChecksPassed, err := t.performPreTerminationHealthChecks(node)
 			if err != nil {
-				return t.transitionToHealing(errors.Wrapf(err, "failed to perform pre-termination health checks, %s is still cordononed", node.Name))
+				t.rm.LogEvent(t.cycleNodeRequest, "PreTerminationHealChecks", "failed to perform pre-termination health checks to %v, err: %v", node.Name, err)
```

Comment:
I can see that the return value of `t.performPreTerminationHealthChecks(node)`, `allHealthChecksPassed`, is still being used as normal. I don't think that it should continue to used as before as, while it returns what you expect right now, I wouldn't expecting a valid return value from a function after it returning an error.

Reply:
I agree with what @dtnyn is saying here. If an error is returned, we cannot continue with the normal flow, the error needs to be handled. What is this fixing? The reason why we transition to healing here is that cycling should fail if the pre-termination trigger fails, for whatever reason. Cyclops should continue with cycling under this condition because that means the workload on the node won't be alerted.